<launch>
    <param name="baseline_bag" value="/home/jaguar/icra_2020/workshop_afternoon_teach.bag"/>
    <param name="experiment_bag" value="/home/jaguar/icra_2020/workshop_fisheye_afternoon_repeat_0.bag"/>
    <param name="icp_iterations" value="300"/> <!-- 300 -->
    <param name="correspondence" value="10"/>
    <param name="ransac" value="0"/>
    <param name="initial_rot" value="40"/>

    <!-- initial_rot 150 zed repeat 0 -->

    <node pkg="pointcloud_align" type="transform_trajectories" name="transform_trajectories" output="screen" />

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find pointcloud_align)/rviz/viz.rviz" />
</launch>